RUFAS/routines/field/crop/crop_management.py

```diff
@@ -67,19 +65,28 @@ def manage_harvest(
         feed_manager : FeedManager
             Instance of the FeedManager that receives harvested crops.
 
+        Returns
+        -------
+        HarvestedCropStorageType | None
+            The mass and nutrional information associated with the harvest's yield, or None if the harvest produced no
+            yield.
+
         """
         self.determine_harvest_index()
 
+        crop = None
         if harvest_op in (HarvestOperation.HARVEST_KILL, HarvestOperation.HARVEST_ONLY):
             self.cut_crop(collected_fraction=self.data.harvest_efficiency)
-            self._store_harvested_crop(time, field_size, feed_manager)
+            crop = self._get_harvested_crop(time, field_size)
```

Comment:
There are a couple other places where it's still crop/crops and I think it would be clearer for it to be harvested_crops whenever possible.
```suggestion
            harvested_crop = self._get_harvested_crop(time, field_size)
```